paddlenlp/datasets/cote.py

```diff
@@ -46,12 +46,42 @@ class Cote(DatasetBuilder):
             },
             'labels': ["B", "I", "O"]
         },
+        'bd': {
+            'url': "https://dataset-bj.cdn.bcebos.com/qianyan/COTE-BD.zip",
+            'md5': "8d87ff9bb6f5e5d46269d72632a1b01f",
+            'splits': {
+                'train': [
+                    os.path.join('COTE-BD', 'train.tsv'),
+                    '4c08ccbcc373cb3bf05c3429d435f608', (0, 1), 1
```

Comment:
Since the format of all Cote datasets are the same. Maybe we dont need `field_indices` the specify data format.

Reply:
done